data/projects/l/lifinance.yaml

```diff
@@ -16,70 +17,267 @@ blockchain:
     tags:
       - deployer
       - eoa
-  - address: "0x2fd525b8b2e2a69d054dccb033a0e33e0b4ab370"
+  - address: "0xCB1e9fA11Edd27098A6B2Ff23cB6b79183ccf8Ee"
```

Comment:
We weren't able to identify contracts deployed from this address. If it's just a project EOA, please remove the `deployer` tag